lib/feature_management.rb

```diff
@@ -159,4 +159,10 @@ def self.idv_by_mail_only?
       outage_status.any_phone_vendor_outage? ||
       outage_status.phone_finder_outage?
   end
+
+  # Whether to use the valid Authn Context Classrefs that include
+  # the newest ACR values
+  def self.use_semantic_authn_contexts?
+    !!IdentityConfig.store.dig(:feature_valid_authn_contexts_semantic_enabled)
```

Comment:
[suggestion] i think this is a little easier to reason about, but i don't feel that strongly about it!

```suggestion
IdentityConfig.store.feature_valid_authn_contexts_semantic_enabled ? true : false
```

Reply:
Fixed by splitting the difference: I kept the `#dig` method because it makes tracing the symbol `:feature_valid_authn_contexts_semantic_enabled` much easier, but instead of using the double bang, I converted to a ternary.